### PR TITLE
Update versions of Logback and Logstash

### PR DESCRIPTION
**Pull request by bspradling8**
Due to a breaking change in `logback-core` here: https://github.com/qos-ch/logback/commit/65a48e835363f31834f221132402308a93d5b843. I'd like to update the versions of these dependencies.

### CODE REVIEW

**Comment by bspradling8** [comment]
Hey @devshorts, are you able to look at this?

File: src/test/java/com/godaddy/logging/logstash/LogstashTests.java [hunk 1]
```diff
@@ -513,11 +513,9 @@ public void test_map() throws IOException {
         OutputStreamAppender<ILoggingEvent> appender = (OutputStreamAppender<ILoggingEvent>) testLogger.getAppender("loggingEventCompositeJsonEncoderAppender");
         LoggingEventCompositeJsonEncoder encoder = (LoggingEventCompositeJsonEncoder) appender.getEncoder();
 
-        ByteArrayOutputStream outputStream = new ByteArrayOutputStream();
-        encoder.init(outputStream);
-        encoder.doEncode(listAppender.list.get(0));
+        String encodedValue = new String(encoder.encode(listAppender.list.get(0)));
```
Comment (devshorts) [inline src/test/java/com/godaddy/logging/logstash/LogstashTests.java]: I think you should specify the character format here to be UTF-8

**Comment by devshorts** [comment]
@bspradling8 only change is the character encoding.  I also don't have the keys to do a deployment of this repo though, so hopefully @bcam909 has shared them :)

**Comment by bspradling8** [comment]
@devshorts Updated with the character encoding. Shoot, he hasn't. I'll try and get ahold of Jake.

**Comment by devshorts** [comment]
@bspradling8 you can temporarily while he is out force push this repo into an internal GD github account for your team, and publish a temporary artifact manually to unblock yourself. When @bcam909 comes back you can publish the public version as well. Might not be a bad idea, then you can have an internal/external view of this repo where the items are staged internally first

**Pull request merged**